cli/utils.go

```diff
@@ -59,3 +63,34 @@ func ParseFileType(raw string) string {
 	}
 	return fmt.Sprintf("%s/%s", osLookup[rawOs], archLookup[rawArch])
 }
+
+func parseBillingAddress(address string) (*apppb.BillingAddress, error) {
+	if address == "" {
+		return nil, errors.New("address is empty")
+	}
+	addr := &apppb.BillingAddress{}
+	splitAddress := strings.Split(address, ",")
+	if len(splitAddress) != 4 && len(splitAddress) != 5 {
+		return nil, errors.Errorf("address: %s does not follow the format: line1, line2, city, state, zipcode", address)
+	}
+
+	currIndex := 0
+
+	addr.AddressLine_1 = splitAddress[currIndex]
+	currIndex++
+
+	if len(splitAddress) == 4 {
+		// if its only 4 lines long that means that there is no line2
+		currIndex++
+	} else {
+		addr.AddressLine_2 = &splitAddress[currIndex]
+		currIndex++
+	}
+
+	addr.City = splitAddress[currIndex]
+	currIndex++
+	addr.State = splitAddress[currIndex]
+	currIndex++
+	addr.Zipcode = splitAddress[currIndex]
+	return addr, nil
```

Comment:
[style, optional] ik repeated code in this but might be more readable to do something like this: 
```suggestion
	if len(splitAddress) == 4 {
		return &apppb.BillingAddress{
		    Address_Line_1: splitAddress[0],
		    City: splitAddress[1],
		    State: splitAddress[2],
		    Zipcode: splitAddress[3],
		}, nil
	} 
	return &apppb.BillingAddress{
		    Address_Line_1: splitAddress[0],
		    Address_Line_2: splitAddress[1],
		    City: splitAddress[2],
		    State: splitAddress[3],
		    Zipcode: splitAddress[4],
		}, nil
```

Reply:
yea thats what i had before, ill revert back to that 👍🏼